src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let temp = 0;
```

Comment:
this is not a temporary value

```suggestion
  let counter = 0;
```